AvaliadorDisciplinas.FRONTEND/src/pages/CourseProfile/CourseProfile.module.css

```diff
@@ -0,0 +1,166 @@
+.container {
```

Comment:
![image](https://user-images.githubusercontent.com/54844612/160636290-72d4b726-ec17-4996-8235-4b52fa974fae.png)
Sugestão para o scroll: como eu comentei na reunião de manhã, talvez o scroll ficasse melhor fora do content, assim:
![image](https://user-images.githubusercontent.com/54844612/160636440-bdfbfc5c-82bf-43a3-bf37-e6ef8447db94.png)
Eu coloquei o código de como ficaria isso mudando poucas linhas de css, basicamente é tirar o `overflow-y` do `.content` e subir pro `.container`, ainda no `.container` você setaria o `width` para 100% e daria padding lateral pro content não ocupar a tela inteira. Depois você tira o `height: 30rem`do `.content` para que ele possa ocupar o tamanho que for necessário de acordo com o conteúdo dele e um `margin-top: 5rem;` para ele ficar separado do `Header`. Aí se quiser mudar o estilo do scroll é só trocar a classe de `.content` para `.container` na estilização de scroll no final desse arquivo css.

Reply:
modificado